profiles/base.yaml

```diff
@@ -1850,6 +1836,34 @@ almanac_no_use_scripts:
 - corn-maze
 - burgle
 
+# List of room titles or room ids or regular expressions
+# that if matches the current room then almanac script won't run.
+almanac_no_use_rooms:
+  - !ruby/regexp /\b(Carousel|Vault|Item Registration Office|Registrar's Office)\b/
```

Comment:
Why not an array with 4 items in it?

Reply:
I think it was because as string literals then they must match the room name exactly. This matches rooms that contain those words since it's a regex match.

Reply:
Is that a concern we have?

Reply:
It's handy, yes, otherwise we have to list out each vault room for every town you might be in because the town is in the room's title.

For example, `[Fang Cove, Carousel Chamber]` or `[Crossing, Carousel Booth]`

Reply:
And if you're asking "why do the string literals have to match the room title exactly?", I had that same question when this paradigm was being introduced 1-2 years ago and the developer at the time was insistent that it match the room name exactly or use regex. So, here we are